include/maths/CBoostedTreeImpl.h

```diff
@@ -120,14 +120,73 @@ class MATHS_EXPORT CBoostedTreeImpl final {
     using TNodeVec = std::vector<CNode>;
     using TNodeVecVec = std::vector<TNodeVec>;
 
+    //! \brief Holds the parameters associated with the different types of regulariser
+    //! terms available.
+    template<typename T>
+    class CRegularization final {
+    public:
+        //! Set the multiplier of the tree size regularizer.
```

Comment:
I guess it is "multiplier of the tree size penalty". Same also below.

Reply:
+1

Reply:
See [f63c228](https://github.com/elastic/ml-cpp/pull/686/commits/f63c2289883ea053fdf643ec38e19a7885a97c56)